vehicle_name: 'e100_2'

######## path generation
goal_x_set: 6 # tracking distance
goal_y_set: 0 # tracking offset
max_track_size: 1000
goal_x_set_behind: -6

######## steer control
wheelbase: 12.56 # for e100
steer_ratio: 9172.0 #180 / 3.14 * 16 * 10 = 9172, adjust smaller to avoid oscillation
min_preview_distance: 5.0
max_preview_distance: 12.0
min_preview_distance_for_search: 3.5
heading_cost_factor: 1.0

######## speed control
speed_cmd_max: 6.0
speed_cmd_min: 0.0
brake_cmd_max: 6
k_front_distance: 1.0
k_behind_distance: 0.0
c_front_velocity: 0.8
c_behind_velocity: 0.0
k_speed_output: 0.6
k_brake_output: 5.0
consider_behind: false

######## point cloud matching
template_file_list: ['e100_1.ply','e100_2.ply','e200.ply']
behind_template_file_list: ['e100_behind2.ply']
goodness_thres: 0.7
vehicle_frame_id: 'vehicle'
map_frame_id: 'map'
target_frame_id: 'target'

########## tracking filter
pose_error_estimation: 0.2
heading_error_estimation: 0.5
speed_error_estimation: 0.01
yaw_rate_error_estimation: 0.06
speed_drive_error_estimation: 0.5
yaw_rate_drive_error_estimation: 0.1
